Platform/MeteorlakeBoardPkg/AcpiTables/Dsdt/Pep.asl

```diff
@@ -717,14 +717,14 @@ Scope(\_SB)
             // standby state with very limited SW activities
             \_SB.PC00.RP01.DL23()
 #if FixedPcdGetBool(PcdMtlPSSupport) == 1
-            Store(0x14040B, Index(RSTG, 0))
+            Store(0xA100B, Index(RSTG, 0)) // GPIOV2_MTL_SOC_M_GPP_A11
```

Comment:
Late we might be able to use GPIOV2_MTL_SOC_M_GPP_A11 directly. Here the ChipsetID (0xA) is used in the hard-coded value. Actually it should be 1 for PAD GPIOV2_MTL_SOC_M_GPP_A11. maybe ChipsetID is not used in ACPI, but it is good to match with GPIOV2_MTL_SOC_M_GPP_A11.

Reply:
Thanks for pointing it out. Your guess is correct: the ChipsetID part (GPIOV2_PAD_POS_CHIPSETID and GPIOV2_PAD_MASK_CHIPSETID) is not used, which is why 0xA worked. I have revised the patch to address the ChipsetID issue.